pool/postgres.go

```diff
@@ -44,53 +44,66 @@ func InitPostgresDB(host string, port uint32, user, pass, dbName string) (*Postg
 		return nil, errs.DBError(errs.DBOpen, desc)
 	}
 
+	makeErr := func(funcName string, table string, err error) error {
```

Comment:
passing `funcName` as a param is unnecessary - it is already in scope for the body of this func.